packages/mds-audit/middleware/audit-api-version.ts

```diff
@@ -0,0 +1,23 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import { ApiVersionMiddleware } from '@mds-core/mds-api-server'
+import { AUDIT_API_SUPPORTED_VERSIONS, AUDIT_API_DEFAULT_VERSION } from '../types'
+
+export const AuditApiVersionMiddleware = ApiVersionMiddleware(
+  'application/vnd.mds.audit+json',
+  AUDIT_API_SUPPORTED_VERSIONS
```

Comment:
Are these versions coming from a global repository, or are they set on a per-app basis?

Also, @drtyh2o I would like to pick your brain about how the versioning middleware works as well as `res.locals.version`.

Reply:
Versions are specified per API. The middleware handles the version negotiation and stashes the version in res.locals ... Check out the provider spec in mobility-data-specification for an overview on how the versioning works generally. Happy to review the specifics with you.